src/stream-analytics/README.md

```diff
@@ -0,0 +1,181 @@
+    --resource-group groupName \
+    --job-name jobName \
+    --name transformationName \
+    --streaming-units 3 \
+    --transformation-query "SELECT * INTO outputName FROM inputName"
+```
+
+##### Create a Stream Analytics Input
+```
+az stream-analytics input create \
+    --resource-group groupName \
+    --job-name jobName \
+    --name inputName \
+    --type Stream \
+    --datasource @datasource.json \
+    --serialization @serialization.json
+```
+datasource.json contains the following content
+```json
+{
+    "type": "Microsoft.Storage/Blob",
+    "properties": {
+        "storageAccounts": [
+            {
+                "accountName": "someAccountName",
+                "accountKey": "someAccountKey=="
+            }
+        ],
+        "container": "state",
+        "pathPattern": "{date}/{time}",
+        "dateFormat": "yyyy/MM/dd",
+        "timeFormat": "HH"
+    }
+}
+```
+serialization.json contains the following content
+```json
+{
+    "type": "Csv",
+    "properties": {
+        "fieldDelimiter": ",",
+        "encoding": "UTF8"
+    }
+}
+```
+
+##### Test a Stream Analytics Input
+```
+az stream-analytics input test \
+    --resource-group groupName \
+    --job-name jobName \
+    --name inputName
+```
+
+##### Create a Stream Analytics Output
+```
+az stream-analytics output create \
+    --resource-group groupName \
+    --job-name jobName \
+    --name outputName \
+    --datasource @datasource.json \
+    --serialization @serialization.json
+```
+datasource.json contains the following content
+```json
+{
+    "type": "Microsoft.DataLake/Accounts",
+    "properties": {
+        "accountName": "someaccount",
+        "tenantId": "cea4e98b-c798-49e7-8c40-4a2b3beb47dd",
+        "refreshToken": "someRefreshToken==",
+        "tokenUserPrincipalName": "bobsmith@contoso.com",
+        "tokenUserDisplayName": "Bob Smith",
+        "filePathPrefix": "{date}/{time}",
+        "dateFormat": "yyyy/MM/dd",
+        "timeFormat": "HH"
+    }
+}
+```
+serialization.json contains the following content
+```json
+{
+    "type": "Json",
+    "properties": {
+        "encoding": "UTF8",
+        "format": "Array"
+    }
+}
+```
+
+##### Test a Stream Analytics Output
+```
+az stream-analytics output test \
+    --resource-group groupName \
+    --job-name jobName \
+    --name outputName
+```
+
+##### Start a Stream Analytics Job
+```
+az stream-analytics job start \
+    --resource-group groupName \
+    --name jobName
+```
+
+##### Stop a Stream Analytics Job
+```
+az stream-analytics job stop \
+    --resource-group groupName \
+    --name jobName
+```
+
+##### Create a Stream Analytics Function
+```
+az stream-analytics function create \
+    --resource-group groupName \
+    --job-name jobName \
+    --name functionName \
+    --inputs @function_inputs.json \
+    --function-output @function_output.json \
+    --binding @function_binding.json
+```
+function_inputs.json contains the following content
```

Comment:
update to 'inputs' to align with argument name

Reply:
updated